docker-compose.yml

```diff
@@ -107,15 +104,13 @@ services:
     environment:
       VITE_AUTH0_DOMAIN: ${VITE_AUTH0_DOMAIN}
       VITE_AUTH0_CLIENT_ID: ${VITE_AUTH0_CLIENT_ID}
-      VITE_AUTH0_AUDIENCE: ${AUTH0_AUDIENCE}
-      VITE_INFRA_SERVER_URL: ${VITE_INFRA_SERVER_URL}
-      VITE_WORKERS_SERVER_URL: ${REACT_APP_API_URL}
-      VITE_SERVER_URL: ${REACT_APP_API_URL}
+      VITE_AUTH0_AUDIENCE: ${VITE_AUTH0_AUDIENCE}
+      WORKERS_SERVICE_URL: ${WORKERS_SERVICE_URL}
       VITE_BUSINESSID: ${VITE_BUSINESSID}
       VITE_MANAGERID: ${VITE_MANAGERID}
-      VITE_BASE_API_URL: ${VITE_BASE_API_URL}
-      VITE_HTTP: ${VITE_HTTP}
-      REACT_APP_API_URL: ${REACT_APP_API_URL}
+      VIT_WEBSITE_SERVICE_URL: ${VIT_WEBSITE_SERVICE_URL}
+      INFRA_SERVICE_URL: ${INFRA_SERVICE_URL}
+      INVENTORY_SERVICE_URL: ${INVENTORY_SERVICE_URL}
     ports:
```

Comment:
לפי מה שאני יודעת, משתני סביבה בvite חייבים להתחיל בvite_

Reply:
לפי מה שהבנו מנחמי המשתנה היחיד שאמור להתחיל ב VITE זה ה website
תקנתי את הפספוס של ה E

Reply:
https://vitejs.dev/guide/env-and-mode